### ISSUE TITLE
A question about ringloss on minst dataset.

### ISSUE THREAD

**Issue by wangx404** [THUFutureLab/gluon-face#3]
In resources/mnist-euclidean directory, several pictures show how feature distribution changes on different training sets.I'm a little confused about the result of ringloss.

with a higher learning rate of 0.1, ringloss leads to a better result on trainset; but with a lower learning rate of 0.01, it leads to a better result on valset. Why would ringloss lead to such a strange result?

**Comment by haoxintong** [comment]
Hi, 
The picture name of ringloss results doesn't mean `lr=0.1` or `lr=0.01`. The training hyper params are same except lambda in `Loss = L_softmax + lambda * L_ring`. 

With larger lambda, the effect of ring loss will increase, so when training same epochs, the viz reuslt of `lambda=0.1` is more similar to a ring compared to that of `lambda=0.01`.

**Comment by wangx404** [comment]
== yes, you are right. I should see origin paper and your code in advance.
